mix.exs

```diff
@@ -22,9 +22,11 @@ defmodule GuardianDb.Mixfile do
   end
 
   defp _applications(:test), do: [:postgrex, :ecto, :logger]
+  defp _applications(:test_config), do: _applications(:test)
```

Comment:
I don't think this is necessary. The environment should probably still just be `test`.

Reply:
The original reason for this setup was to test the default configuration (no `:token_types` specified) as well as a config file with specified types. I'm not sure if there is a better way to test multiple configurations like this or if this amount of testing is even worth maintaining for this project.